common/src/stack/command/stack/commands/list/host/plugin_basic.py

```diff
@@ -49,6 +49,14 @@ def run(self, args):
 			if row[0] in host_info:
 				host_info[row[0]].extend(row[1:])
 
```

Comment:
I'd suggest moving the attr lookup out of the loop.

```
host_kicks = self.owner.getHostAttrDict(hosts, attr='kickstartable'))
for host in hosts:
  if not self.owner.str2bool(host_kicks[host]['kickstartable']):
  ...etc
```

Reply:
Updated.